CMakeLists.txt

```diff
@@ -214,9 +214,11 @@ if(VELOX_ENABLE_PARQUET)
 endif()
 
 if(VELOX_ENABLE_REMOTE_FUNCTIONS)
```

Comment:
Is this active on one of the CI builds?

Reply:
@assignUser yes, on `linux-adapters`.

Reply:
https://github.com/facebookincubator/velox/blob/main/.circleci/config.yml#L437